tests/integration/targets/postgresql_ext/tasks/postgresql_ext_initial.yml

```diff
@@ -1,9 +1,6 @@
 - name: postgresql_ext - install postgis on Linux
   package: name=postgis state=present
   when: ansible_os_family != "Windows"
-- name: postgresql_ext - install postgis on Windows
```

Comment:
@Andersson007 community.general isn't tested against Windows CI targets, so we are removing this. Is that OK with you?

Reply:
@gundalow , yep, sure

Reply:
(it is probably something obsolete)